functions-introduction/solutions/weather-quarkus/src/test/java/functions/FunctionTest.java

```diff
@@ -0,0 +1,50 @@
+package functions;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+//import org.hamcrest.CoreMatchers;
```

Comment:
Can we remove the commented imports?

Reply:
Pablo yes sure.

Let me remove the commented imports.